src/Chronos.php

```diff
@@ -2574,25 +2558,15 @@ public static function fromNow(Chronos|DateTimeInterface $other): DateInterval|b
      * 1 hour after
      * 5 months after
      *
-     * @param \Cake\Chronos\Chronos|null $other The datetime to compare with.
+     * @param \DateTimeInterface|null $other The datetime to compare with.
      * @param bool $absolute removes time difference modifiers ago, after, etc
      * @return string
      */
-    public function diffForHumans(?Chronos $other = null, bool $absolute = false): string
+    public function diffForHumans(?DateTimeInterface $other = null, bool $absolute = false): string
     {
         return static::diffFormatter()->diffForHumans($this, $other, $absolute);
     }
 
-    /**
-     * Returns the time as a DateTimeImmutable instance.
-     *
-     * @return \DateTimeImmutable
-     */
-    public function toNative(): DateTimeImmutable
-    {
-        return $this->native;
-    }
```

Comment:
This method was **REMOVED** in a **BUGFIX** release?! This change would have broken production releases if our automated testing didn't catch it.

Reply:
The removal was a mistake, we'll get that fixed.